homeassistant/components/homeassistant/triggers/event.py

```diff
@@ -49,6 +64,10 @@ async def async_attach_trigger(
     event_types = template.render_complex(
         config[CONF_EVENT_TYPE], variables, limited=True
     )
+    if any(evt == EVENT_STATE_REPORTED for evt in event_types):
```

Comment:
```suggestion
    if EVENT_STATE_REPORTED in event_types:
```
Its not possible to do a member check?